backend/sending_data/keys.json

```diff
@@ -0,0 +1,3 @@
+{
+    "ADD PRIVATE KEYS JSON HERE BY DOWNLOADING FROM FIREBASE SETTINGS": null
```

Comment:
### keys.json file formatting

Can you add sample private keys here so that we know specifically which variables we need to add to this file?
For example, `apiKey: FIREBASE_API_KEY` inside of the `keys.json` file instead of `"ADD PRIVATE KEYS ~ "`
You can set all the values inside the `keys.json` file to `null` as you did already.